index.html

```diff
@@ -89,6 +89,212 @@ <h1 class="presentation">
+          </div>
+          <div class="work__description">
+            <h2>Multi-Post Stories</h2>
+            <div class="short_desc">
+              <h4>CANOPY</h4>
+              <span></span>
+              <h4>Back end Dev</h4>
+              <span></span>
+              <p>2015</p>
+            </div>
+            <div class="long_desc">
+              <p>
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+              <ul class="work__lang">
+                <li>html</li>
+                <li>css</li>
+                <li>javascript</li>
+              </ul>
+            </div>
+            <button type="button" class="btn btn-enabled">See project</button>
+          </div>
+        </article>
+      </section>
+      <section class="about-me" id="about">
+        <div class="about-me-container">
+          <div class="about-me__title">
+            <h2>About Myself</h2>
+          </div>
+          <div class="about-me__description">
+            <p>
+              Hello I’m a software developer! I can help you build a product ,
+              feature or website Look through some of my work and experience! If
+              you like what you see and have a project you need coded, don’t
+              hestiate to contact me.
+            </p>
+          </div>
+          <a href="#" class="medias">let's connect</a>
+          <ul class="social-media">
+            <li>
+              <a href="#"
+                ><img
+                  src="images/icons/twitter.svg"
+                  alt="twitter"
+                  title="follow me on twitter"
+              /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img
+                  src="images/icons/Linkedin_icon.svg"
+                  alt="linkedin"
+                  title="follow me on linkedin"
+              /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img
+                  src="images/icons/medium.svg"
+                  alt="medium"
+                  title="follow me on"
+              /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img
+                  src="images/icons/github.svg"
+                  alt="github"
+                  title="follow me on github"
+              /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img
+                  src="images/icons/angellist.svg"
+                  alt="angellist"
+                  title="follow me on"
+              /></a>
+            </li>
+          </ul>
+          <button type="button" class="btn btn-enabled modalBtn">
+            Get my resume
+          </button>
+        </div>
+
+        <div class="about-me__tools">
+          <div class="tools prog-languages">
+            <h3>languages</h3>
```

Comment:
- [ ] Please kindly ensure the technology categories ("Languages", "Frameworks" and "Skills") are in a `ul`, each of them in a `li`

```
<ul>
<li>Languages</li>
<li>Frameworks</li>
<li>Skills</li>
</ul>
```